Since XCP-ng 8.1, local SRs are ext4 by default.

We don't know if the local SRs are ext3 are ext4 so simply remove 
the digit and display 'Ext'.

Index: xsconsole-10.1.10/XSConsoleLangFriendlyNames.py
===================================================================
--- xsconsole-10.1.10/XSConsoleLangFriendlyNames.py
+++ xsconsole-10.1.10/XSConsoleLangFriendlyNames.py	2020-03-12 11:53:04.170073770 +0100
@@ -599,7 +599,7 @@
     'Label-SR.SRTypes-egenera' : 'Egenera Virtual Storage',
     'Label-SR.SRTypes-egeneracd' : 'Egenera Virtual DVD Drive',
     'Label-SR.SRTypes-equal' : 'Dell EqualLogic',
-    'Label-SR.SRTypes-ext' : 'Ext3',
+    'Label-SR.SRTypes-ext' : 'Ext',
     'Label-SR.SRTypes-iso' : 'ISO',
     'Label-SR.SRTypes-local' : 'Local',
     'Label-SR.SRTypes-lvm' : 'LVM',
